tests/unit/test_queue_service.py

```
@@ -629,30 +629,30 @@ async def test_get_queue_stats_default_queue(self, supabase_queue):
         mock_metrics.total_messages = 0
         mock_metrics.newest_msg_age_sec = 0
         mock_metrics.oldest_msg_age_sec = 0
-        
+
         supabase_queue.queue.metrics = AsyncMock(return_value=mock_metrics)
-        
+
         stats = await supabase_queue.get_queue_stats()
```

Ruff (0.12.2)

Local variable `stats` is assigned to but never used

Remove assignment to unused variable `stats`

(F841)